saltstack/final/salt/cis-controls/playbooks/selinux.yml

```diff
@@ -13,9 +13,8 @@
         - selinux
         - CB-27855
 
-    - name: Allow httpd_t (e.g nginx) to connect to port 7070
+    - name: Allow httpd_t (e.g nginx) to connect to downstream service ports
       shell: | 
-        semanage port -a -t http_port_t -p tcp 7070
         setsebool -P httpd_can_network_connect on
```

Comment:
(Not related to this PR)
All the selinux related commands here could probably be moved to salt states. As far as I can see they are only here because there is an OS version check for executing this playbook, but that check could also be done in a salt state.
@kkubina might be able to tell if there is any reason not to move them to the salt states.

Reply:
Technically we could also install our policy modules through Ansible to have the SELinux related configuration in one place. I'll probably ask Kristof if he is okay with me consolidating these SELinux related configurations in one place, as they are a bit scattered right now. 😄